docs/iris/src/whatsnew/contributions_2.1/newfeature_2018-May-08_repr-html.txt

```diff
@@ -0,0 +1,2 @@
+* Added ``repr_html`` functionality to the :class:`~iris.cube.Cube` to provide
```

Comment:
We will want this to have a screenshot when it comes to cutting the release.

Reply:
We could use the image above?